forms-flow-web/src/components/Form/Item/Submission/Item/View.js

```diff
@@ -54,7 +54,7 @@ const View = React.memo((props) => {
     <div className="container row task-container">
       <div className="main-header">
         <h3 className="task-head"> {form.title}</h3>
-        {showPrintButton && form?._id ? (
+        {(showPrintButton && form?._id && !props.submission.draftDetailStatusCode ) ? (
```

Comment:
Need to update the condition with some other reliable property to check if the item is a draft or not

Reply:
Yup, Draft Id and draft enable env check would be better?

Reply:
ok